website/frontend/src/components/App/App.tsx

```diff
@@ -24,6 +26,15 @@ import 'assets/scss/common.scss';
 import 'assets/scss/index.scss';
 import { AppContextProvider } from '../../contexts/AppContext/AppContextProvider';
 
+// alert configuration
+const options = {
+  position: positions.BOTTOM_CENTER,
+  timeout: 5000,
+  containerStyle: {
+    zIndex: 999,
```

Comment:
could you move it to enum with zIndex values? causing its hard to manage with z-indexes later